atrium-core/build.gradle.kts

```diff
@@ -1,3 +1,7 @@
+plugins {
+    id("build-logic.published-kotlin-multiplatform")
+}
```

Comment:
we loose the syntax support if we apply it via `subprojects` in the root build.gradle.kts right?

Reply:
The key issue with `subprojects` is that it is hard to opt-out of this apply.

For instance, if I want to add a project that is "Kotlin compiler plugin", then I do not need `kolin-multiplatform` there.

At the same time, if every project declares its plugins in its `plugins {...}` block, then it is easier to understand the build logic.

Reply:
that's a fair point